versioned_docs/version-1.2/apis-clients/java-client-examples/process-deploy.md

```diff
@@ -15,7 +15,7 @@ description: "Let's analyze the prerequisites and code to deploy a process using
 
 ## ProcessDeployer.java
 
-[Source on GitHub](https://github.com/camunda-cloud/zeebe/tree/develop/samples/src/main/java/io/camunda/zeebe/example/process/ProcessDeployer.java)
+[Source on GitHub](https://github.com/camunda-community-hub/camunda-8-examples/blob/main/zeebe-client-plain-java/src/main/java/io/camunda/zeebe/example/process/ProcessDeployer.java)
```

Comment:
There is another link on this page that links to the location of the `demoProcess.bpmn`. Please also modify this one.